### DISCUSSION TITLE
QEMU error when trying to boot up Win 11 VM after restarting

### DISCUSSION THREAD

**Discussion by qwerty523** [utmapp/UTM#6146] [Windows]
I set up Windows 11 yesterday, and it worked pretty much okay. That night, I shut down my computer. The next morning, I woke up and tried to load it again, but it threw this error:
<img width="552" alt="Screenshot 2023-12-03 at 9 17 51 AM" src="https://github.com/utmapp/UTM/assets/98148464/88a302a9-8c42-4638-a5ac-b1e39d6bbb38">
I'm guessing it either got corrupted when I shut the computer down or that a critical setting changed for no reason. Do I need to set up a new VM with this iso file, download an entirely new one, or just change a setting?

**Configuration**  
* UTM Version: 4.4.4
* macOS Version: Sonoma 14.1
* Mac Chip (Intel, M1, ...): Intel Core i5

**Comment by CocoTech1861** [comment]
I had this same issue, still no solution 3 months later

**Comment by Tinus016** [comment]
try selecting your iso again, it worked for me 
<img width="906" alt="UTM_ISO_Select" src="https://github.com/utmapp/UTM/assets/60196891/b71c389e-f9a9-495b-a688-51b3aefc8ace">

**Comment by CocoTech1861** [comment]
@Tinus016 That worked, thank you!
